server/db/migrate/20160926115601_add_cfme_multiregion_fields_to_deployment.rb

```diff
@@ -0,0 +1,8 @@
+class AddCfmeMultiregionFieldsToDeployment < ActiveRecord::Migration
+  def change
+    rename_column :fusor_deployments, :cfme_address, :cfme_rhv_address
```

Comment:
This may be a document issue, but by renaming the old column, anyone who upgrades from 1.0 to 1.1 and deployed CFME to OpenStack will have some incorrect info on the old deployment since it will now be in cfme_rhv_* instead of cfme_osp_*.  You probably want to find out if that's ok.  

If not, we could also move the data to the correct column using the migration up/down:

1. Create new columns cfme_rhv_address, cfme_rhv_hostname, cfme_osp_address, cfme_osp_hostname
2. Copy data from cfme_address and cfme_hostname over to the correct columns based on cfme_install_loc
3. Destroy the old columns cfme_address and cfme_hostname.

You'd also have to do the reverse in the down.